packages/swap/src/utils/intercept.ts

```diff
@@ -0,0 +1,26 @@
+import { Request, Response, NextFunction } from 'express';
+import prisma from '../client';
+
+export const getLastChainTrackingUpdateTimestamp = async () => {
+  const latestChainTracking = await prisma.chainTracking.findFirst({
+    orderBy: {
+      updatedAt: 'desc',
+    },
+  });
+
+  if (!latestChainTracking) {
+    return undefined;
+  }
+
+  return latestChainTracking.updatedAt;
+};
+
+export const stalenessCheck = async (_: Request, res: Response, next: NextFunction) => {
+  const lastUpdateTimestamp = await getLastChainTrackingUpdateTimestamp();
+
+  if (lastUpdateTimestamp) {
+    res.header('X-Last-Update', lastUpdateTimestamp.toUTCString());
```

Comment:
should we keep or remove this? I see no harm in having it, more visibility.

Reply:
fine for me to keep it but would call it `X-Last-Statechain-Update` then to keep the name consistent

Reply:
should be `X-CF-${string`

Reply:
oh no cloudflare

Reply:
`X-Flip-${string}`?

Reply:
`X-Chainflip-...` ?